src/util/profile/profile.ts

```diff
@@ -100,14 +101,15 @@ class ProfileImpl implements Profile {
   }
 }
 
-const validApp = /^([a-zA-Z0-9-_]{3,100})\/([a-zA-Z0-9-_]{3,100})$/;
+const validApp = /^([a-zA-Z0-9-_.]{1,100})\/([a-zA-Z0-9-_.]{1,100})$/;
```

Comment:
Yes, the requirements for users and apps have changed recently, this reflects the latest validations on the server.